repose-aggregator/core/repose-core/src/test/scala/org/openrepose/core/spring/NextVersionTimebombTest.scala

```diff
@@ -77,6 +77,8 @@ class NextVersionTimebombTest extends FunSpec with Matchers with TestFilterBundl
        *     Before doing so, verify that this behavior is not useful.
        *
        * 11. Remove the population of X-Auth-Token-Key from Keystone v2 Filter.
+       *
+       * 12. Allow Jetty (in a Valve deployment) to handle errors when sendError is called on the response.
```

Comment:
I'm not certain we want to do that actually. I think we may want to change the behavior of an empty body, but I don't think we ever want valve to give an html response.

Reply:
Fair enough, I'll drop this note.